web/tests/lynxkite.ts

```diff
@@ -1,5 +1,4 @@
 // Shared testing utilities.
-// TODO: This is being migrated from test-lib.js. We will clean it up at the end.
```

Comment:
It's not too bad now! I'm not sure all the methods are actually used but I couldn't find a way to find dead code. There are two TODOs left. Both are reasonable, but not urgent.